src/styles/index.css

```diff
@@ -0,0 +1,67 @@
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+}
+
+.stopwatch__minuteshand,
+.stopwatch__secondshand {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform-origin: 50% 100%;
+  transform: translate(-50%, -100%) rotate(0deg);
+}
+
+.stopwatch__minuteshand {
+  height: 20vmin;
+  width: 3vmin;
+  background-color: #0700ff;
+  animation: step-minutes 600s steps(60) infinite;
+}
+
+.stopwatch__secondshand {
+  height: 38vmin;
+  width: 1.5vmin;
+  background-color: #2c8000;
+  animation: spin-seconds 10s linear infinite;
```

Comment:
The `animation` property for `.stopwatch__secondshand` is set to `10s`, which matches the `.speed-up` class duration. If the intention is to have a different speed for the `.speed-up` class, consider adjusting the duration.